tally_ho/apps/tally/views/super_admin.py

```diff
@@ -360,6 +391,99 @@ def get(self, *args, **kwargs):
             tally_id=tally_id))
 
 
+class DuplicateResultTrackingView(LoginRequiredMixin,
+                                  mixins.GroupRequiredMixin,
+                                  mixins.TallyAccessMixin,
+                                  TemplateView):
+    group_required = groups.SUPER_ADMINISTRATOR
+    template_name = "super_admin/duplicate_result_tracking.html"
+
+    def get(self, *args, **kwargs):
+        tally_id = kwargs['tally_id']
+
+        return self.render_to_response(self.get_context_data(
+            duplicate_results=get_result_form_with_duplicate_results(
+                tally_id=tally_id),
+            tally_id=tally_id))
+
+
+class DuplicateResultFormView(LoginRequiredMixin,
+                              mixins.GroupRequiredMixin,
+                              mixins.TallyAccessMixin,
+                              SuccessMessageMixin,
+                              TemplateView):
+    group_required = groups.SUPER_ADMINISTRATOR
+    template_name = "super_admin/duplicate_result_form.html"
+    success_url = "duplicate-result-tracking"
+
+    def get(self, *args, **kwargs):
+        tally_id = kwargs['tally_id']
+        barcode = kwargs['barcode']
+        ballot_id = kwargs['ballot_id']
+        result_form = ResultForm.objects.get(barcode=barcode)
+        results = Result.objects.filter(result_form=result_form.id)
+
+        return self.render_to_response(self.get_context_data(
+            results_form_duplicates=get_result_form_with_duplicate_results(
+                ballot=ballot_id,
+                tally_id=tally_id),
+            results=results,
+            tally_id=tally_id,
+            ballot_id=ballot_id,
+            header_text="Form " + str(barcode)))
+
+    def post(self, *args, **kwargs):
+        tally_id = kwargs['tally_id']
+        ballot_id = kwargs['ballot_id']
+        post_data = self.request.POST
+        results_form_duplicates = get_result_form_with_duplicate_results(
+            ballot=ballot_id,
+            tally_id=tally_id)
+
+        if 'duplicate_reviewed' in post_data:
+            for results_form_duplicate in results_form_duplicates:
```

Comment:
I'd also note that we *could* do this without a loop and just a single DB call, *but* since we don't expect this array to be that large, it's fine as is